modules/add-pod-namespace-to-sensor-and-admission-control.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * upgrade/upgrade-from-45.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="add-pod-namespace-to-sensor-and-admission-control_{context}"]
+= Add `POD_NAMESPACE` to `sensor` and `admission-control` deployments
```

Comment:
```suggestion
= Adding `POD_NAMESPACE` to `sensor` and `admission-control` deployments
```
[use gerunds](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#assemblymodule-titles-and-section-headings) in assembly & module titles

Reply:
Oh, sorry - one other thing - we can't use backticks in module and assembly headings: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#module-file-metadata

Reply:
Done, and squashed the commits 👍